logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -15,6 +15,21 @@ def close
   end
 end
 
+class DummyInputGenerator < LogStash::Inputs::Base
```

Comment:
we install the generator plugin as part of the core plugins need for test, https://github.com/elastic/logstash/blob/master/rakelib/default_plugins.rb#L113-L121 any reason not to redefined here?   you know I would like our make our test plugin free, only use mocks/stubs/dummy (pick your name), but if we require it for now I think it would be better to stick to one usage. what do you think?